crates/forge_analyzer/src/checkers.rs

```diff
@@ -1112,7 +1119,7 @@ impl<'cx> Dataflow<'cx> for PermissionDataflow {
                     interp.bitbucket_permission_resolver,
                     interp.bitbucket_regex_map,
                 ),
-                _ => unreachable!("Invalid intrinsic function type"),
+                _ => (&PermissionHashMap::new(), &HashMap::<String, Regex>::new()),
```

Comment:
Could you add the rest of the cases to make this match exhaustive? So we don't forget to update this in the event that we add other services/intrinsics.

Reply:
No, I mean just filling out the match arm with the rest of the variants of `IntrinsicName`. There should be a `rust_analyzer` code action for doing this.

Also, you should install [rust-analyzer](https://rust-analyzer.github.io/) if you haven't already.